spread.yaml

```diff
@@ -89,7 +89,7 @@ prepare: |
       cat "$tempfile"
       exit 1
   fi
-  tests.pkgs install snapd libc6-dev git make
+  tests.pkgs install snapd libc6-dev git make fdisk jq
```

Comment:
This is more a question for @upils than a comment about this PR: should we be including `fdisk` in the snap?

Reply:
It already is in the snap :D https://github.com/canonical/imagecraft/blob/8f5fab001d223d57f87d94d8db8161de4984ee1a/snap/snapcraft.yaml#L89